db/backup_download.go

```diff
@@ -131,3 +135,12 @@ func isDir(path string) bool {
 
 	return s.IsDir()
 }
+
+func getLastSuccessfulBackup(backups []scalingo.Backup) (string, error) {
+	for _, backup := range backups {
+		if backup.Status == scalingo.BackupStatusDone {
+			return backup.ID, nil
+		}
+	}
+	return "", errgo.New("Can't find any successful backup")
```

Comment:
Error message starts with a lower case in Go (https://www.notion.so/scalingooriginal/Golang-Best-Practice-de47c3b8d6c24c2e9b70c7b9c2f36660#76e817f954bb46ed85adb08bfd850f83)
```suggestion
	return "", errgo.New("can't find any successful backup")
```